### ISSUE TITLE
Feature Proposal: Email verification

### ISSUE THREAD

**Issue by winston-yallow** [fedora-infra/flask-oidc#72]
It seems like there is no way to enforce email verification since `OIDC_REQUIRE_VERIFIED_EMAIL` was deprecated.

I propose to add a decorator `require_verified_email()` that only grants access if the mail is verified.

**Comment by winston-yallow** [comment]
To be clear, it is possible to manually check `session['oidc_auth_profile']['email_verified']`, but the proposal would integrate this better into a typical flask workflow.

**Comment by abompard** [comment]
Yeah, it sounds reasonable, could you write the PR? I'll review it.

**Comment by davidtavarez** [comment]
@winston-yallow can you explain the flow? what happens when the email is not verified? a redirect? a 403?
